### ISSUE TITLE
Invite Daniel Bojar for a talk in webinar series

### ISSUE THREAD

**Issue by ReneRanzinger** [glygener/glygen-issues#608]
I hope you are doing well. At the Beilstein meeting we talked about our GlyGen seminar series and the possibility for you to give a talk about your work. We have a monthly GlyGen Webinar series, usually on the first Tuesday of the month, to which we invite people to give a talk about their research, approx. 30-40 minutes to help our team understand more about their work.  The webinars are also publicized more generally through various listservs (SfG, eX-twitter etc.) and have been drawing reasonably broad audiences. 

Usually, the seminar is recorded, and the recordings are released on our YouTube channel. We've had Harry Brumer, Rebekah Gundry, Chris West, Stacy Malakar, Bob Haltiwanger, and Lingjun Li give talks. And a talk from Manfred Wuhrer is coming up later this month.

I would like to invite you to give a talk at our seminar and I wonder whether you might be available in the first half of December. The talks are usually at 10:00 AM US Eastern Time.  The topic can be anything of interest to you.

The talk is usual followed by another meeting to discuss collaboration and data exchange between our GlyGen project the speakers. One topics that is of interest to us are the integration of data from your curated database into GlyGen. Another is our aim to provide “ML ready” datasets and I hope you and your group can provide feedback and guidance. Of course any interest from your side on GlyGen data and suggestions are very welcome.
 
I've copied Jeet Vora onto this email.  He is part of Raja Mazumder's data team at George Washington University and he is the organizer of the GlyGen seminar series. 
 
Let me know if you have any questions or if you prefer a date in beginning of 2024 over December.

**Comment by ReneRanzinger** [comment]
Dear René,

thanks for your mail & for this invitation!

Sure, the first half of December still looks quite flexible in my calendar and I would be happy to give a talk at the GlyGen Webinar series. And yes, the discussion that is planned for afterwards seems like a very good idea. I will be sure to also collect ideas for this within my group:-)

Let me know what you need when from my side.
Thanks a lot & looking forward,
Daniel

**Comment by jeet-vora** [comment]
December 12

 1. Your profile picture
  --> Attached
2. LinkedIn profile or departmental profile link
  --> https://www.linkedin.com/in/daniel-bojar-418647b2/
3. Title of your talk
  --> From Complexity to Clarity: Algorithmic Exploration of Glycans
4. Three sentences abstract about your talk
  --> Navigating the intricate world of glycans requires innovative approaches to untangle their complexity and reveal their functional roles. By merging advanced computational techniques with cutting-edge glycoscience research, we have developed a set of innovative tools that provide deeper insights into glycan properties and functions, shedding light on their roles in health and disease. As will be presented in the talk, we combined these approaches in our hub program, glycowork, including glycan sequence analysis, glycomics data analysis, biosynthetic analyses, and many more.
